src/averageAges.js

```diff
@@ -15,11 +15,10 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men =  people.filter(isMen => century ? isMen.sex === 'm'
+    && Math.ceil(isMen.died / 100) === century : isMen.sex === 'm')
```

Comment:
You need to create callbacks and use them here with ternary operator.
Remember, callback is a separate function.
Check Vova's comments again.